test/service/test_manifest.py

```diff
@@ -1273,6 +1273,32 @@ def test_manifest_content_disposition_header(self):
                         actual_cd = query.params['response-content-disposition']
                         self.assertEqual(expected_cd, actual_cd)
 
+    @unittest.skipIf(not config.enable_replicas,
+                     'The format is replica-based')
```

Comment:
This will have to be changed in the PR for https://github.com/DataBiosphere/azul/issues/6062